Go/shortUrl.go

```diff
@@ -0,0 +1,58 @@
+/**
+ * ShortURL: Bijective conversion between natural numbers (IDs) and short strings
+ * Licensed under the MIT License (https://opensource.org/licenses/MIT)
+ *
+ * ShortURL::encode() takes an ID and turns it into a short string
+ * ShortURL::decode() takes a short string and turns it into an ID
+ *
+ * Features:
+ * + large alphabet (51 chars) and thus very short resulting strings
+ * + proof against offensive words (removed 'a', 'e', 'i', 'o' and 'u')
+ * + unambiguous (removed 'I', 'l', '1', 'O' and '0')
+ **/
+package ShortURL
+
+import (
+	"fmt"
+	"strings"
+)
+
+const (
+	Alphabets = "23456789bcdfghjkmnpqrstvwxyzBCDFGHJKLMNPQRSTVWXYZ-_"
+	Base      = len(Alphabets)
+)
```

Comment:
Do we need to export `Alphabets` and `Base`?

Reply:
All the other implementations (at least PHP,Java) have them public, so for the compatibility sake so Go too?

I would imagine it would be nice to have them as public, maybe some web servers need to do some white listing, or dynamic routing based on them?